pkgs/development/interpreters/perl/default.nix

```diff
@@ -206,11 +198,19 @@ in {
     sha256 = "0vs0wwwlw47sswxaflkk4hw0y45cmc7arxx788kwpbminy5lrq1j";
   };
 
+  # Maint version
+  perl532 = common {
+    perl = pkgs.perl532;
+    buildPerl = buildPackages.perl532;
+    version = "5.32.0";
+    sha256 = "1d6001cjnpxfv79000bx00vmv2nvdz7wrnyas451j908y7hirszg";
+  };
+
   # the latest Devel version
   perldevel = common {
     perl = pkgs.perldevel;
     buildPerl = buildPackages.perldevel;
-    version = "5.32.0-RC0";
-    sha256 = "02i6n1xa4j0ksp014yy8q0j7scjcy5mr0yd4iash2ryrrfv5yw5k";
+    version = "5.32.0";
```

Comment:
```suggestion
    version = "5.33.0";
```

Reply:
@volth A bit unsure how to best proceed. 

Would you like me to force push to https://github.com/volth/nixpkgs-windows/tree/perl532 ? 
Or open a new PR?